hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/BlockManagerImpl.java

```diff
@@ -223,20 +223,26 @@ public void deleteBlocks(List<BlockGroup> keyBlocksInfoList)
     // TODO: track the block size info so that we can reclaim the container
     // TODO: used space when the block is deleted.
     for (BlockGroup bg : keyBlocksInfoList) {
+      List<DeletedBlock> deletedBlocks = bg.getAllBlocks();
```

Comment:
we can have only latest list with translated with size = 0 and repeatedSize=0 if data is not available during upgrade.